RELEASES.md

```diff
@@ -1,3 +1,132 @@
+Version 1.80 (2024-07-25)
+==========================
+
+<a id="1.80-Language"></a>
+
+Language
+--------
+- [Document maximum allocation size](https://github.com/rust-lang/rust/pull/116675/)
+- [Allow zero-byte offsets and ZST read/writes on arbitrary pointers](https://github.com/rust-lang/rust/pull/117329/)
+- [Support C23's variadics without a named parameter](https://github.com/rust-lang/rust/pull/124048/)
+- [Stabilize exclusive_range_pattern](https://github.com/rust-lang/rust/pull/124459/)
+- [Guarantee layout and ABI of `Result` in some scenarios](https://github.com/rust-lang/rust/pull/124870)
+- [Warn (or error) when `Self` ctor from outer item is referenced in inner nested item](https://github.com/rust-lang/rust/pull/124187/)
+- [Turn non-structural-const-in-pattern lints into hard errors](https://github.com/rust-lang/rust/pull/124661/)
+- [Make `WHERE_CLAUSES_OBJECT_SAFETY` lint a regular object safety violation](https://github.com/rust-lang/rust/pull/125380/)
+
+
+<a id="1.80-Compiler"></a>
+
+Compiler
+--------
+- [Update cc crate to v1.0.97 allowing additional specter mitigations on MSVC targets](https://github.com/rust-lang/rust/pull/124892/)
+- [Allow field reordering on types marked `repr(packed(1))`](https://github.com/rust-lang/rust/pull/125360/)
+- [Detect unused structs even when implementing private traits](https://github.com/rust-lang/rust/pull/122382/)
+- [Add a lint against never type fallback affecting unsafe code](https://github.com/rust-lang/rust/pull/123939/)
+- [Add `--print=check-cfg` to get the expected configs](https://github.com/rust-lang/rust/pull/124320/)
+- [Add `rustfmt` cfg to well known cfgs list](https://github.com/rust-lang/rust/pull/124742/)
+- [Support `Result<T, E>` across FFI when niche optimization can be used](https://github.com/rust-lang/rust/pull/122253/)
+- [Disallow cast with trailing braced macro in let-else](https://github.com/rust-lang/rust/pull/125049/)
+- [Expand `for_loops_over_fallibles` lint to lint on fallibles behind references.](https://github.com/rust-lang/rust/pull/125156/)
+- [self-contained linker: retry linking without `-fuse-ld=lld` on CCs that don't support it](https://github.com/rust-lang/rust/pull/125417/)
+- [Convert `proc_macro_back_compat` lint to an unconditional error.](https://github.com/rust-lang/rust/pull/125596/)
+- [Do not parse CVarArgs (`...`) as a type in trait bounds](https://github.com/rust-lang/rust/pull/125863/)
+- [Correctly handle type aliases during orphan check](https://github.com/rust-lang/rust/pull/117164/)
+- Improvements to LLDB formatting [#124458](https://github.com/rust-lang/rust/pull/124458) [#124500](https://github.com/rust-lang/rust/pull/124500)
+- [For the wasm32-wasip2 target default to PIC and do not use `-fuse-ld=lld`](https://github.com/rust-lang/rust/pull/124858/)
+- [Adjust 64-bit ARM data layouts for LLVM update](https://github.com/rust-lang/rust/pull/124813/)
```

Comment:
This is only for the upcoming LLVM 19, and not something that I think needs announcing anyway.
```suggestion
```